File: Hash or Objects/Question 6/ae_count.rb
def ae_count(str)
    count = {"a"=>0, "e"=>0}
  
    str.each_char do |char|
      if (char == "a" || char == "e")
        count[char] += 1
      end
    end
  
    return count
  end
  
  puts ae_count("everyone can program") #=> {"a"=>2, "e"=>3}
  puts ae_count("keyboard") #=> {"a"=>1, "e"=>1}

File: Hash or Objects/Question 6/ae_count.js
function ae_count(str){
    let count = {'a':0, 'e': 0};
    let word = str.split('');
  
    for(let i =0; i<word.length; i++){
      if(word[i] === 'a' || word[i] === 'e'){
        count[word[i]] += 1;
      }
    }
    return count;
  }
  
  
  console.log(ae_count('everybody hates me'));